translate/v3/translate_text.go

```diff
@@ -55,15 +59,16 @@ func translateText(w io.Writer, projectID string, sourceLang string, targetLang
 	if err != nil {
 		return fmt.Errorf("TranslateText: %v", err)
 	}
+	// [END translate_v3_translate_text_2]
 
+	// [START translate_v3_translate_text_2]
```

Comment:
Copy/paste error? Should this block be `translate_v3_translate_text_3` as in the Node sample?

Reply:
Whoops, thanks!!